### DISCUSSION TITLE
rdx-proxy has incorrect timestamp

### DISCUSSION THREAD

**Discussion by VitaliiTrubchaninovUbisoft** [rancher-sandbox/rancher-desktop#5144] [General]
Is it normal that rdx-proxy image has timestamp "292 years ago" in "docker image ls"?
It messes up with our tools because timestamp value can not be converted to C# DateTime object.
Created field is invalid as valid values are between -62135596800 and 253402300799, inclusive.
 {
        "Containers": -1,
        "Created": -62167219200,
        
        "Id": "sha256:3e9d7c4baf0523db53580bb9c16fc3c2e90470eee3ffb1b3a7fb68feb76052e5",
        "Labels": null,
        "ParentId": "",
        "RepoDigests": null,
        "RepoTags": [
            "ghcr.io/rancher-sandbox/rancher-desktop/rdx-proxy:latest"
        ],
        "SharedSize": -1,
        "Size": 5103616,
        "VirtualSize": 5103616
    }

**Comment by jandubois** [comment]
> Is it normal that rdx-proxy image has timestamp "292 years ago"

Yes, it is "normal", but not really correct, so we should fix it. I've turned your entry here into an issue at #5145.

**Reply by VitaliiTrubchaninovUbisoft** [reply]
Thank you.

**Comment by jandubois** [comment]
I've created PR #5149 to drop the creation date from the metadata. Not sure if your tool will handle it, but it does get rid of the "292 years ago" silliness:

```
$ docker image ls ghcr.io/rancher-sandbox/rancher-desktop/rdx-proxy:latest
REPOSITORY                                          TAG       IMAGE ID       CREATED   SIZE
ghcr.io/rancher-sandbox/rancher-desktop/rdx-proxy   latest    01fb3e91e58c   N/A       5.1MB

$ docker image ls ghcr.io/rancher-sandbox/rancher-desktop/rdx-proxy:latest --format json | jq
{
  "Containers": "N/A",
  "CreatedAt": "0000-12-31 15:47:32 -0812 LMT",
  "CreatedSince": "",
  "Digest": "<none>",
  "ID": "01fb3e91e58c",
  "Repository": "ghcr.io/rancher-sandbox/rancher-desktop/rdx-proxy",
  "SharedSize": "N/A",
  "Size": "5.1MB",
  "Tag": "latest",
  "UniqueSize": "N/A",
  "VirtualSize": "5.095MB"
}
```

See also https://github.com/docker/cli/issues/2047.
